bench-cargo-miri/slice-chunked/src/main.rs

```diff
@@ -0,0 +1,26 @@
+//! This is a small example using slice::chunks, which creates a very large Tree Borrows tree.
+//! Thanks to ##3837, the GC now compacts the tree, so this test can be run in a reasonable time again.
+//! The actual code is adapted from tiny_skia, see https://github.com/RazrFalcon/tiny-skia/blob/master/src/pixmap.rs#L121
+//! To make this benchmark demonstrate the effectiveness, run with MIRIFLAGS="-Zmiri-tree-borrows -Zmiri-provenance-gc=100"
```

Comment:
How long does this benchmark take to run with default flags (i.e., Stacked Borrows)?

Reply:
About 450 millis on my machine. For TB, if you tune the GC properly (i.e. set it to run every 100 BBs), you can get to about 530 millis. With the normal GC setting, it takes about 3 seconds, without the GC about 50.